### ISSUE TITLE
Default for new gutter color option is not bg+

### ISSUE THREAD

**Issue by blueyed** [junegunn/fzf#1556]
The man page says that the default for the gutter is `bg+` (https://github.com/junegunn/fzf/blob/ff951341c993ed84ad65344e496e122ee3dddf67/man/man1/fzf.1#L258), but that does not appear to be the case with `--color=16,bg+:18`.

Related code might be: https://github.com/junegunn/fzf/blob/ff951341c993ed84ad65344e496e122ee3dddf67/src/tui/tui.go#L414

Specifying it explicitly works.

I can see that it is good to have it default to `bg+` (the previous behavior, which I am missing now), but maybe making it default to `bg` would be better (and result in what you get with `-1` then?).

fzf 0.18.0

**Comment by junegunn** [comment]
Oh it looks to be a bug with `16`:

```sh
# bg+ == gutter
fzf --color=bg+:1
fzf --color=dark,bg+:1
fzf --color=light,bg+:1

# bg+ != gutter
fzf --color=16,bg+:1
```

**Comment by junegunn** [comment]
This was fixed in 8ae96774df202497c7e73d24e10c51e621eabddf
